src/main/java/com/karankumar/bookproject/backend/entity/Book.java

```diff
@@ -47,6 +47,9 @@
     private Genre genre;
 
     private Integer seriesPosition;
+    
+    @NotNull
+    private int edition;
```

Comment:
This should be an `Integer` (apologies, I know I said it should be an `int` in the issue description)